File: Pattern8.c
// 1
// 2  3
// 4  5  6
// 7  8  9  10
// 11 12 13 14 15
#include <stdio.h>

void main()
{
    int n;
    printf("Enter the row number:");
    scanf("%d", &n);
    int count = 0;
    for (int i = 1; i <= n; i++)
    {
        for (int j = 1; j <= i; j++)
        {
            printf("%d ", ++count);
        }
        printf("\n");
    }
}

File: Pattern8.go
package main

import "fmt"

func main(){
	var n int;
	fmt.Printf("Enter the number:")
	fmt.Scanln(&n)
	var count int = 0
	for i:= 1; i <= n; i++{
		for j:= 1; j <= i; j++ {
			count++
			fmt.Printf("%d " ,count);
		}
		fmt.Printf("\n")
	}
}